src/server/api/routes/create-user-roles.routes.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+const createUserRolesController = require('../controllers/create-user-roles.controller');
+
+/**
+ * @swagger
+ * /user_roles:
+ *  post:
+ *    summary: Create a user role
+ *    description:
+ *      Will create a specific role for a user.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: user roles
+ *        description: The user roles to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - fk_role_id
+ *            - fk_user_id
+ *          properties:
+ *            fk_role_id:
+ *              type: integer
+ *            fk_user_id:
+ *              type: integer
+ *    responses:
+ *      201:
+ *        description: User roles created
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.post('/', (req, res) => {
+  createUserRolesController
+    .createModule(req.body)
```

Comment:
Same here. You have to call the controller function. Also try to use `req, res, next`.

Reply:
You also need to resolve the conflicts below.

Reply:
got it 👍